src/blocks/RulesBlock.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+
+function RulesModuleHeader() {
+  return (
+    <div className="rules-header">
+      <h3>📌 Pinned Rules</h3>
```

Comment:
Make the title of this header configurable later on :) I think on the main page it will likely be "Pinned Rules", but I can imagine that internal realm pages could have longer lists of secondary rules, each with their own title.